src/main/java/codeu/controller/RegisterServlet.java

```diff
@@ -48,6 +48,18 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
       throws IOException, ServletException {
 
     String username = request.getParameter("username");
+    String password = request.getParameter("password");
+    boolean isAttack = false;
+    
+    if (username.contains(";") || username.contains("'") || username.contains("\"")) {
```

Comment:
It's possible any of these could be a typo. Might it be more meaningful to perhaps look for SQL specific keywords (i.e. SELECT)?